moduleroot/Makefile.erb

```diff
@@ -76,6 +76,15 @@ golden-diff: commodore_args = -f tests/$(instance).yml --search-paths ./dependen
 golden-diff: .compile ## Diff compile output against the reference version. Review output and run `make gen-golden golden-diff` if this target fails.
 	@git diff --exit-code --minimal --no-index -- tests/golden/$(instance) compiled/
 <%- end -%>
+<%- if @configs['feature_kubevalValidate'] -%>
+
+.PHONY: validate
+validate: commodore_args = -f tests/$(instance).yml --search-paths ./dependencies
+validate: .compile ## Validate the resulting Kubernetes objects using kubeval.
+	@echo
+	@echo
+	$(KUBEVAL_DOCKER) $(KUBEVAL_ARGS) -d compiled/
+<%- end -%>
```

Comment:
resulting from the discussion, I would not call it `validate` since that's not a formal validation but rather just K8s-specific linting, thus name it `lint_kubeval` or so.
```suggestion
<%- if @configs['feature_kubevalLint'] -%>

.PHONY: lint_kubeval
lint_kubeval: commodore_args = -f tests/$(instance).yml --search-paths ./dependencies
lint_kubeval: .compile ## Lint the resulting Kubernetes objects using kubeval (ignores CRDs!)
	@echo
	@echo
	$(KUBEVAL_DOCKER) $(KUBEVAL_ARGS) -d compiled/
<%- end -%>
```